pkg/apis/planetscale/v2/labels.go

```diff
@@ -40,6 +40,8 @@ const (
 	TabletPoolNameLabel = LabelPrefix + "/" + "pool-name"
 	// TabletIndexLabel is the key for identifying the index of a Vitess tablet within its pool.
 	TabletIndexLabel = LabelPrefix + "/" + "tablet-index"
+	// BackupScheduleLabel is the key for identifying to which VitessBackupSchedule a Job belongs to.
+	BackupScheduleLabel = LabelPrefix + "/" + "backup-schedule"
```

Comment:
Could this also be accomplished with an owner reference?

Reply:
That might be a little neater, since if you delete a backup schedule, you'd probably also want to delete any currently running pods.

Reply:
The `VitessBackupSchedule` is already the owner of its own jobs, which makes it an owner of the pods that are created by the jobs. When deleting a backup schedule all the associated pods and jobs are also removed.

```
$ kubectl get pods                                                                                                                                                                                          
NAME                                                         READY   STATUS              RESTARTS   AGE
...
example-vbsc-every-minute-94d12263-1717100280-9cscg          0/1     ContainerCreating   0          27s
...

$  kubectl get vitessbackupschedule                                                                                                                                                                           
NAME                                 AGE
example-vbsc-every-minute-94d12263   90s

$ kubectl delete vitessbackupschedule example-vbsc-every-minute-94d12263                                                                                                                                     
vitessbackupschedule.planetscale.com "example-vbsc-every-minute-94d12263" deleted

$ kubectl get pods                                                                                                                                                                                           
NAME                                                         READY   STATUS        RESTARTS      AGE
...
example-vbsc-every-minute-94d12263-1717100280-9cscg          1/1     Terminating   1 (31s ago)   67s
...

$ kubectl get pods                                                                                                                                                                                         
NAME                                                         READY   STATUS    RESTARTS      AGE
example-90089e05-vitessbackupstorage-subcontroller           1/1     Running   0             2m30s
example-commerce-x-x-vtbackup-init-c6db73c9                  0/1     Error     2 (26s ago)   2m26s
example-commerce-x-x-zone1-vtorc-c13ef6ff-5855667dbc-d5dcl   1/1     Running   1 (73s ago)   2m27s
example-etcd-faf13de3-1                                      1/1     Running   0             2m30s
example-etcd-faf13de3-2                                      1/1     Running   0             2m30s
example-etcd-faf13de3-3                                      1/1     Running   0             2m30s
example-vttablet-zone1-0790125915-4e37d9d5                   3/3     Running   0             2m27s
example-vttablet-zone1-2469782763-bfadd780                   3/3     Running   0             2m27s
example-vttablet-zone1-2548885007-46a852d0                   3/3     Running   0             2m27s
example-zone1-vtctld-1d4dcad0-646b8c9c77-vl69s               1/1     Running   1 (73s ago)   2m29s
example-zone1-vtgate-bc6cde92-569bbcf4df-lslph               1/1     Running   1 (72s ago)   2m27s
vitess-operator-6f87b84f88-6wjlt                             1/1     Running   0             2m33s
```

Reply:
> Could this also be accomplished with an owner reference?

I am not 100% sure of what you mean by that.